src/main/java/org/opendevstack/provision/services/JenkinsPipelineAdapter.java

```diff
@@ -61,6 +61,8 @@
 
   public static final String PROJECT_ID_KEY = "PROJECT_ID";
 
+  public static final String OPTION_KEY_GIT_SERVER_URL = "GIT_SERVER_URL";
```

Comment:
@michaelsauter  is there a hidden knob in the webhook proxy host, to cater to this name?